README.md

```diff
@@ -1,13 +1,99 @@
-# GBFS Validator
+# GBFS-validator
+A General [Bikeshare Feed Specification](https://github.com/NABSA/gbfs) dataset validator
 
-> Validate GBFS endpoint
+# Introduction
 
-## How to contribute
+The Canonical GBFS Validator is a tool to check the conformity of a GBFS feed against the [official specification](https://github.com/NABSA/gbfs/blob/master/gbfs.md).
+It validates feeds up to version 2.2.
+This tool is built using the [JSON Schemas](https://github.com/NABSA/gbfs/blob/master/gbfs.md).
 
-Setup everything with
+![interface](https://user-images.githubusercontent.com/63653518/133172186-6842537b-f34c-42bd-bd5a-48ec65a0d8c3.png)
 
-`yarn install`
+Currently, the schemas are reproduced per folder in [gbfs-validator/tree/master/gbfs-validator/schema](https://github.com/fluctuo/gbfs-validator/tree/master/gbfs-validator/schema).
 
-then
+Questions? Please open an issue or reach out to MobilityData on the gbfs slack channel!
 
-`yarn run dev`
+# Run the app
+The validator is developed to be used “online” (hosted with a lambda function).
+1. Open gbfs-validator.netlify.com/
+2. Enter the feed’s auto-discovery URL
+3. If needed, select the version. If not specified, the validator will pick the version mentioned in the `gbfs.json` file
+4. Select file requirement options (free-floating or docked)
+5. Click the “Valid me” button, and see the validation results below
+
+# Validation rules
+The validation rules are listed in [RULES.md](/RULES.md)
+Have a suggestion for a new rule? Open an issue!
+
+# Contributing
+We welcome contributions to the project! Please check out our [Contribution guidelines](/CONTRIBUTING.md) for details.
+
+# Build the project: Web server install procedure
```

Comment:
I'm not a big fan of this part because it's a bit specific and subject to deprecation.
Instead, we could add a Dockerfile to simplify startup (of course Docker should be installed) or/and just add link to official install procedure (for yarn https://classic.yarnpkg.com/en/docs/install and node https://nodejs.org/en/download/, prefered instead https://github.com/nvm-sh/nvm)

I checked, it's working well on github codespace (https://github.com/features/codespaces) (paying functionnality)

Reply:
Okay, can you update this part with this proposition?

Reply:
I made a commit. Can you review.
After that, it's ok for me. Anyway we can update it later if something is not clear